platform/platform-configuration/src/main/java/org/codice/ddf/configuration/config/mapping/PlatformConfigurationConfigMappingProvider.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.configuration.config.mapping;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.codice.ddf.config.ConfigService;
+import org.codice.ddf.config.mapping.ConfigMapping;
+import org.codice.ddf.config.mapping.ConfigMappingException;
+import org.codice.ddf.config.mapping.ConfigMappingProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * NOTE: This is an example of a Config Mapping Provider that only provides default values. These
+ * values will be overridden by upstream projects.
+ */
+public class PlatformConfigurationConfigMappingProvider implements ConfigMappingProvider {
+
+  private static final Logger LOGGER =
+      LoggerFactory.getLogger(PlatformConfigurationConfigMappingProvider.class);
+
+  private static final String BACKGROUND = "background";
```

Comment:
Do some of these constants already exist in the project by any chances? if so we should reference those

Reply:
Yes, fixed.